docs/guides/alert-manager-configuration.md

```diff
@@ -0,0 +1,103 @@
+`cortextool --help-long`
+
+>  1. Create `.yaml` file to specify the configuration for AlertManager notification characteristic.  
+
+>>> The following is `amconfig.yml`, an example of a configuration for AlertManager to send notification to a Slack channel
+
+```
+global:
+  resolve_timeout: 10m
+  slack_api_url: '<slack incoming web hook defined for your slack workspace>'
+route:
+  group_wait: 10s
+
+# When the first notification was sent, wait 'group_interval' to send a batch
+# of new alerts that started firing for that group
+  group_interval: 1m 
+
+# If an alert has successfully been sent, wait 'repeat_interval' to
+# resend them.
+  repeat_interval: 10m
+
+  receiver: slack_receiver
+# All the above attributes are inherited by all child routes and can
+# overwritten on each.
+
+  routes:
+    - receiver: slack_receiver
+      match_re:
+        severity: critical|warning
+      continue: true
+
+receivers:
+- name: slack_receiver
+  slack_configs:
+  - send_resolved: true
+    channel: '#<slack channel where to send the alert>'
+
+# The following just describe what to put in the alert
+    title: |-
+      [{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}] {{ if .CommonLabels.alertname }}{{ .CommonLabels.alertname }}{{ else }}Multiple Alerts Open{{ end }}{{ if .CommonLabels.node }} for {{ .CommonLabels.node }}{{ else if .CommonLabels.nodename }} for {{ .CommonLabels.nodename }}{{ end }}
+            text: >-
+              {{ range .Alerts -}}
+              *Alert:* {{ .Annotations.title }}
+              *Description:* {{ .Annotations.description }}
+                 • *Environment:* {{ if .Labels.environment }}`{{ .Labels.environment }}`{{ else }}`N/A`{{ end }}
+                 • *Hostname:* {{ if .Labels.node }}`{{ .Labels.node }}`{{ else if .Labels.nodename }}`{{ .Labels.nodename }}`{{ else }}`N/A`{{ end }}
+                 • *Condition:* {{ if .Labels.condition }}`{{ .Labels.condition }}`{{ else }}`N/A`{{ end }}
+                 • *Current value:* {{ if .Annotations.current_value }}`{{ .Annotations.current_value }}`{{ else }}`N/A`{{ end }}
+                 • *Opened at:* {{ if .StartsAt }}`{{ .StartsAt }}`{{ else }}`N/A`{{ end }}
+                 • *Closed at:* {{ if eq .Status "resolved" }}`{{ .EndsAt }}`{{ else }}`Unresolved, alert still active.`{{ end }}
+              *Details:*
+                {{ range .Labels.SortedPairs }} • *{{ .Name }}:* `{{ .Value }}`
+                {{ end }}
+              {{ end }}
+```
+
+>>> [Full Example of Alertmanager configuration](https://github.com/prometheus/alertmanager/blob/master/doc/examples/simple.yml)
+
+>>> [Example on how to setup Slack](https://grafana.com/blog/2020/02/25/step-by-step-guide-to-setting-up-prometheus-alertmanager-with-slack-pagerduty-and-gmail/#:~:text=To%20set%20up%20alerting%20in,to%20receive%20notifications%20from%20Alertmanager.) to support receiving AlertManager notification
```

Comment:
Let's simplify the link to `https://grafana.com/blog/2020/02/25/step-by-step-guide-to-setting-up-prometheus-alertmanager-with-slack-pagerduty-and-gmail` only.